layouts/partials/home-services.html

```diff
@@ -15,20 +15,20 @@ <h2 class="display-4 text-black">Services</h2>
                 <div class="icon d-flex align-items-end"><img src="/img/icons/startup.svg" alt="..." class="img-fluid"></div>
                 <h3 class="h4">Cloud Kickstart</h3>
                 <p class="font-weight-light">A full cloud platform up and running in days, including CI/CD, monitoring and support.</p>
-                <a class="" href="/services/kickstart">Cloud Kickstart »</a>
+                <a class="" href="/services/kickstart">Explore Cloud Kickstart »</a>
               </div>
-           </div>  
- 
+           </div>
+
             <div class="col-lg-4">
               <div class="box text-center">
                 <div class="icon d-flex align-items-end"><img src="/img/icons/developer.svg" alt="..." class="img-fluid"></div>
-                <h3 class="h4">Cloud Consultancy</h3>
-                <p class="font-weight-light">Long term or short term engagements: our consultants can help you with cloud related projects.</p>
-                <a class="" href="/services/consultancy">Hire a consultant »</a>
+                <h3 class="h4">Cloud Coach</h3>
+                <p class="font-weight-light">Our Cloud Coaches help you grow your startup into a healthy Cloud-native company without breaking the bank.</p>
```

Comment:
Cloud -> cloud lijkt me mooier, maar ik weet niet of dat past qua consistentie met oog op de rest van de website.

Reply:
Ga ik even checken. Als het qua consistentie beter is pas ik het aan. 👍